packages/connector-alipay-native/src/constant.ts

```diff
@@ -29,7 +29,7 @@ export const defaultMetadata: ConnectorMetadata = {
     en: 'Alipay',
     'zh-CN': '支付宝',
   },
-  logo: './logo.png',
+  logo: 'https://gist.githubusercontent.com/darcyYe/31bc893a0a305dc43cf831bf0b14f0fc/raw/faf985d3fbeed88180b8f3cb709892320d66ae45/alipay.svg',
```

Comment:
I'm curious where we can find these svg source files. Are they uploaded to one of your private repos?

Reply:
Currently, they are on my GitHub gist but are publicly accessible. Due to the network accessibility issue in some areas, @wangsijie will put them on some other server, here is the [issue](https://linear.app/silverhand/issue/LOG-2594/serve-connectors-logo).

Reply:
ok